x-pack/plugins/enterprise_search/public/applications/workplace_search/views/overview/overview.test.tsx

```diff
@@ -5,61 +5,48 @@
  * 2.0.
  */
 
+import '../../../__mocks__/shallow_useeffect.mock';
 import './__mocks__/overview_logic.mock';
 import { mockActions, setMockValues } from './__mocks__';
 
 import React from 'react';
 
-import { shallow, mount } from 'enzyme';
-
-import { Loading } from '../../../shared/loading';
-import { ViewContentHeader } from '../../components/shared/view_content_header';
+import { shallow } from 'enzyme';
 
 import { OnboardingSteps } from './onboarding_steps';
 import { OrganizationStats } from './organization_stats';
 import { Overview } from './overview';
 import { RecentActivity } from './recent_activity';
 
 describe('Overview', () => {
-  describe('non-happy-path states', () => {
-    it('isLoading', () => {
```

Comment:
Mostly just indenting changes in this file due to deleting the happy/non-happy path blocks - I recommend viewing the diff with whitespace changes off.

Also as a heads up, I'm removing most `isLoading` tests because they're now a simple/basic prop pass-through to the underlying page template, which [already has its own unit tests for](https://github.com/elastic/kibana/blob/66c9d801d5cdcd48f0d5ee0472a337763547fed4/x-pack/plugins/enterprise_search/public/applications/shared/layout/page_template.test.tsx#L44-L45) for loading state.